config/data_scrubber_test.go

```diff
@@ -43,6 +46,157 @@ func setupDataScrubberWildCard(t *testing.T) *DataScrubber {
+		{[]string{"1-password --config=12345"}, []string{"1-password", "--config=********"}},
+		{[]string{"java kafka password 1234"}, []string{"java", "kafka", "password", "********"}},
+		{[]string{"agent", "password:1234"}, []string{"agent", "password:********"}},
+		{[]string{"agent password:1234"}, []string{"agent", "password:********"}},
+	}
+}
+
+func setupInsensitiveCmdlines() []testCase {
+	return []testCase{
+		{[]string{"spidly", "--debug_port=2043"}, []string{"spidly", "--debug_port=2043"}},
+		{[]string{"agent", "start", "-p", "config.cfg"}, []string{"agent", "start", "-p", "config.cfg"}},
+		{[]string{"p1", "--openpassword=admin"}, []string{"p1", "--openpassword=admin"}},
+		{[]string{"p1", "-openpassword", "admin"}, []string{"p1", "-openpassword", "admin"}},
+		{[]string{"java -openpassword 1234"}, []string{"java -openpassword 1234"}},
+		{[]string{"java -open_password 1234"}, []string{"java -open_password 1234"}},
+		{[]string{"java -passwordOpen 1234"}, []string{"java -passwordOpen 1234"}},
+		{[]string{"java -password_open 1234"}, []string{"java -password_open 1234"}},
+		{[]string{"java -password1 1234"}, []string{"java -password1 1234"}},
+		{[]string{"java -password_1 1234"}, []string{"java -password_1 1234"}},
+		{[]string{"java -1password 1234"}, []string{"java -1password 1234"}},
+		{[]string{"java -1_password 1234"}, []string{"java -1_password 1234"}},
+		{[]string{"agent", "1_password:1234"}, []string{"agent", "1_password:1234"}},
+		{[]string{"agent 1_password:1234"}, []string{"agent 1_password:1234"}},
+	}
+}
+
+func setupCmdlinesWithWildCards() []testCase {
+	return []testCase{
+		{[]string{"spidly", "--befpass=2043", "onebefpass", "1234", "--befpassCustom=1234"},
+			[]string{"spidly", "--befpass=********", "onebefpass", "********", "--befpassCustom=1234"}},
+		{[]string{"spidly --befpass=2043 onebefpass 1234 --befpassCustom=1234"},
+			[]string{"spidly", "--befpass=********", "onebefpass", "********", "--befpassCustom=1234"}},
+		{[]string{"spidly   --befpass=2043   onebefpass   1234   --befpassCustom=1234"},
+			[]string{"spidly", "", "", "--befpass=********", "", "", "onebefpass", "", "", "********", "", "", "--befpassCustom=1234"}},
+
+		{[]string{"spidly", "--afterpass=2043", "afterpass_1", "1234", "--befafterpass_1=1234"},
+			[]string{"spidly", "--afterpass=********", "afterpass_1", "********", "--befafterpass_1=1234"}},
+		{[]string{"spidly --afterpass=2043 afterpass_1 1234 --befafterpass_1=1234"},
+			[]string{"spidly", "--afterpass=********", "afterpass_1", "********", "--befafterpass_1=1234"}},
+		{[]string{"spidly   --afterpass=2043   afterpass_1   1234   --befafterpass_1=1234"},
+			[]string{"spidly", "", "", "--afterpass=********", "", "", "afterpass_1", "", "", "********", "", "", "--befafterpass_1=1234"}},
+
+		{[]string{"spidly", "both", "1234", "-dd_both", "1234", "bothafter", "1234", "--dd_bothafter=1234"},
```

Comment:
Yes, I did enable `gofmt` on the atomBeautify package. I think that it's using one more tabbing because i'm splitting the initialization of the struct in two lines

Reply:
I see what's happening. This is really not readable though. Can you fix this to be like:

```
return []testCase{
 {
   []string{"spidly", "--befpass=2043", "onebefpass", "1234", "--befpassCustom=1234"},
   []string{"spidly", "--befpass=********", "onebefpass", "********", "--befpassCustom=1234"},
 },
 ...
}
```